ui/components/app/whats-new-popup/whats-new-popup.js

```diff
@@ -85,6 +85,9 @@ function getActionFunctionById(id, history) {
         url: ZENDESK_URLS.LEDGER_FIREFOX_U2F_GUIDE,
       });
     },
+    21: () => {
```

Comment:
1. Every time `getActionFunctionById` gets called, the whole object gets rebuilt, even though we're only using one.  I think you could `useMemo` or `useCallback` or move the definition outside of that function.  I actually don't know how frequently `getActionFunctionById` is called though, so maybe it doesn't matter **_that_** much.
2. You could probably have a default case that takes `id` and sets `updateViewedNotifications({ id: true });`.  Almost all of them include this, and some of them are only this.

Reply:
We only run this when there are new notifications, so i don't think it would matter too much.